buildSrc/src/main/java/org/elasticsearch/gradle/testfixtures/TestFixturesPlugin.java

```diff
@@ -136,7 +136,7 @@ public void apply(Project project) {
 
         if (dockerComposeSupported() == false) {
             project.getLogger()
-                .warn(
+                .debug(
```

Comment:
Can we use `info` instead? Debug level logging is verbose to the point of being nearly useless and `info` is not logged by default either unless you ask with `--info`.

Reply:
Done.